smileidentity/web-client
# [ehn] Refactor SmartCamera and SmartFileUpload

# Summary
This extracts several pages into separate components so that they can be displayed or hidden based on the partners preference.

# Todo

- [x] Refactor ID capture instruction screens into a component
- [x] Refactor ID capture screen into a component
- [x] Refactor ID capture review screens into a component
- [x] Allow partners to hide screens from the id capture flow
- [x] Refactor selfie capture instructions into a component


# Testing
- Install dependencies localling `npm install`
- Run `npm run storybook`
- Open http://localhost:6006/ to few the stories
- Under the DocumentCapture test out the different scenarios and verify the outcome
   - Capture With instruction screen
   - Capture without instructions screen
   - Capture without back of id screen
